plugins/GSdx/GSCrc.h

```diff
@@ -109,6 +111,11 @@ class CRC
 		Onimusha3,
 		PiaCarroteYoukosoGPGakuenPrincess,
 		RadiataStories,
+		RatchetAndClank,
+		RatchetAndClank2,
+		RatchetAndClank3,
+		RatchetAndClank4,
+		RatchetAndClank5,
 		RedDeadRevolver,
 		ResidentEvil4,
 		RozenMaidenGebetGarden,
```

Comment:
Ditto. (See comment above)

Reply:
I prefer to have this list under alphabetical order. Looks clean.